akka-docs/src/main/paradox/serialization.md

```diff
@@ -302,3 +286,23 @@ It must still be possible to deserialize the events that were stored with the ol
 * [Akka-kryo by Roman Levenstein](https://github.com/romix/akka-kryo-serialization)
 
 * [Twitter Chill Scala extensions for Kryo (based on Akka Version 2.3.x but due to backwards compatibility of the Serializer Interface this extension also works with 2.4.x)](https://github.com/twitter/chill)
+
+### Verification
+
+Normally, messages sent between local actors (i.e. same JVM) do not undergo serialization. For testing, sometimes, it may be desirable to force serialization on all messages (both remote and local). If you want to do this in order to verify that your messages are serializable you can enable the following config option:
+
+@@snip [SerializationDocSpec.scala](/akka-docs/src/test/scala/docs/serialization/SerializationDocSpec.scala) { #serialize-messages-config }
+
+Certain messages can be excluded from verification by extending the marker @scala[trait]@java[interface]
+`akka.actor.NoSerializationVerificationNeeded` or define a class name prefix in configuration
+`akka.actor.no-serialization-verification-needed-class-prefix`.
+
+If you want to verify that your `Props` are serializable you can enable the following config option:
+
+@@snip [SerializationDocSpec.scala](/akka-docs/src/test/scala/docs/serialization/SerializationDocSpec.scala) { #serialize-creators-config }
+
+@@@ warning
+
+We recommend having these config options turned on **only** when you're running tests. Turning these options on in production is pointless, as it would negatively impact the performance of local message passing without giving any gain.
+
+@@@
```

Comment:
I'd hoped we could not only de-emphasize but deprecate the feature now that we have the `SerializationTestKit`

Reply:
`SerializationTestKit` is only for Typed (so far) and it doesn't help detecting messages that don't have a serializer.